git/references.go

```diff
@@ -0,0 +1,80 @@
+package git
+
+import (
+	"github.com/gitql/gitql/sql"
+
+	"gopkg.in/src-d/go-git.v4"
+	"gopkg.in/src-d/go-git.v4/core"
+	"io"
```

Comment:
gofmt and group deps (first standard library, then internal dependencies, then external dependencies).